src/components/Pages/EventsPage/HorizontalFilterBox.js

```diff
@@ -270,6 +270,57 @@ class HorizontalFilterBox extends Component {
     this.props.search(e);
     putSearchTextFilterInURL(this.props, e.target.value);
   };
+
+//changes text of button depending on the page that its on
+  renderAddButton = () => {
+      if (this.props.user) {
+
+        var ButtonText = ""
+        var ModalType = ""
+
+        if (window.location.pathname.includes("testimonial")) {
```

Comment:
I think this should use the "Type" prop.

Reply:
Please see latest commit